packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -165,11 +165,13 @@ module.exports = {
       }
     ]
   },
+  // @remove-on-eject-begin
   // Point ESLint to our predefined config.
   eslint: {
-    configFile: path.join(__dirname, 'eslint.js'),
+    configFile: path.join(__dirname, '../.eslintrc'),
     useEslintrc: false
   },
+  // @remove-on-eject-end
```

Comment:
We copy the `.eslintrc` to the ejected project, so we don't need to specify a custom `configFile` path or to disable the `.eslintrc`. I think people prefer the rc files for configuration in an ejected project (see #410).